test_scripts/LowVoltage/015_Low_Voltage_and_Wake_Up_absence_resumption_LIMITED_hmi_level_in_case_existed_app_in_limited.lua

```diff
@@ -0,0 +1,71 @@
+---------------------------------------------------------------------------------------------------
+-- In case:
+-- 1. Mobile app is in LIMITED HMILevel
+-- 2. App has some persistent data that can be resumed
+-- 2. Mobile app is disconnected during the time frame of 30 sec (inclusive) before "SDL_LOW_VOLTAGE" unix signal from HMI
+-- 3. Mobile app registers with the same hashID during 30 sec. after "WAKE_UP" unix signal in the same ignition cycle
+-- 4. there is application currently in LIMITED
```

Comment:
with mobile projection, there can be multiple apps in LIMITED

Reply:
@yang1070 This was an original requirement for resumption functionality that is transferred to Low Voltage